data.h

```diff
@@ -281,6 +282,48 @@ struct CsvColumnInfo {
 
 class Table {
  public:
+  // Creates a table with a single column with uniformly distributed values.
+  static std::unique_ptr<Table> GenerateUniform(const size_t generate_num_values,
```

Comment:
I would make this a separate method in evaluation_utils.h or so  – I don't think this method belongs to the Table class.

Reply:
Sounds good.